test/unit_tests/test_translation.py

```diff
@@ -315,6 +390,273 @@ def test_translate_operation_with_unique_params(
         )
 
 
+OQC_PULSE_PROPERTIES = json.dumps(
+    {
+        "braketSchemaHeader": {
+            "name": "braket.device_schema.pulse.pulse_device_action_properties",
+            "version": "1",
+        },
+        "supportedQhpTemplateWaveforms": {},
+        "ports": {
+            "channel_15": {
+                "portId": "channel_15",
+                "direction": "tx",
+                "portType": "port_type_1",
+                "dt": 5e-10,
+            },
+            "channel_11": {
+                "portId": "channel_11",
+                "direction": "tx",
+                "portType": "port_type_1",
+                "dt": 5e-10,
+            },
+        },
+        "supportedFunctions": {},
+        "frames": {
+            "q0_drive": {
+                "frameId": "q0_drive",
+                "portId": "channel_15",
+                "frequency": 4.6e9,
+                "centerFrequency": 4360000000.0,
+                "phase": 0.0,
+                "associatedGate": None,
+                "qubitMappings": [0],
+                "qhpSpecificProperties": None,
+            },
+            "q0_second_state": {
+                "frameId": "q0_second_state",
+                "portId": "channel_15",
+                "frequency": 4.5e9,
+                "centerFrequency": 4360000000.0,
+                "phase": 0.0,
+                "associatedGate": None,
+                "qubitMappings": [0],
+                "qhpSpecificProperties": None,
+            },
+            "q1_drive": {
+                "frameId": "q0_drive",
+                "portId": "channel_11",
+                "frequency": 4.6e9,
+                "centerFrequency": 4360000000.0,
+                "phase": 0.0,
+                "associatedGate": None,
+                "qubitMappings": [0],
+                "qhpSpecificProperties": None,
+            },
+            "q1_second_state": {
+                "frameId": "q0_second_state",
+                "portId": "channel_11",
+                "frequency": 4.5e9,
+                "centerFrequency": 4360000000.0,
+                "phase": 0.0,
+                "associatedGate": None,
+                "qubitMappings": [0],
+                "qhpSpecificProperties": None,
+            },
+        },
+        "supportsLocalPulseElements": False,
+        "supportsDynamicFrames": True,
+        "supportsNonNativeGatesWithPulses": True,
+        "validationParameters": {
+            "MAX_SCALE": 1.0,
+            "MAX_AMPLITUDE": 1.0,
+            "PERMITTED_FREQUENCY_DIFFERENCE": 1.0,
```

Comment:
Are those values correct? In particular permitted freq difference? Shouldnt it be `1e9`?

Reply:
Just double checked against the actual parameters, and it's correct. These parameters come straight from OQC, so maybe OQC uses GHz conventionally while braket uses Hz?

Reply:
🙈 
thanks for checking!
one more thing to keep in mind then 😆